modules/core/src/main/scala/ch/epfl/bluebrain/nexus/kg/core/resources/Payload.scala

```diff
@@ -0,0 +1,32 @@
+package ch.epfl.bluebrain.nexus.kg.core.resources
+
+import io.circe.Json
+
+/**
+  * Enumeration type for payloads.
+  */
+sealed trait Payload extends Product with Serializable
```

Comment:
I'm wondering if it would be better to define this as:
`final case class Payload(mediaType: String, value: String)`
thoughts?

Reply:
Why this should be better? I prefer to have a more strict type. In encoding and decoding is easier. We tried the other approach already